### ISSUE TITLE
 'IAppBuilder' does not contain a definition for 'UseUmbracoBackAuthentication' 

### ISSUE THREAD

**Issue by terryclancy** [Shazwazza/UmbracoIdentity#32]
Hi UmbracoIdentity folk,

As suggested at  https://github.com/Shazwazza/UmbracoIdentity/wiki/Startup-Configuration

I am trying to call the following in my Startup Class:

            //Ensure owin is configured for Umbraco back office authentication
            app.UseUmbracoBackAuthentication();

but that causes the following error:

error CS1061: 'IAppBuilder' does not contain a definition for 'UseUmbracoBackAuthentication' and no extension method 'UseUmbracoBackAuthentication' accepting a first argument of type 'IAppBuilder' could be found (are you missing a using directive or an assembly reference?)

Intellisence confirms that UseUmbracoBackAuthentication does not appear as a method of "app".

Note that prior to that line the following line works fine:

          app.ConfigureUserManagerForUmbracoMembers<UmbracoApplicationMember>();

Any suggestions as to what he problem is with app.UseUmbracoBackAuthentication(); would be very much appreciated.

Thanks 

Terry Clancy

ClanceZ

**Comment by terryclancy** [comment]
Nevermind,

I just realized I am running Umbraco 7.3 but pre-requisite is 7.3.4.  I will update and report back.  In the mean time you can ignore this issue.

Terry Clancy

ClanceZ

**Comment by terryclancy** [comment]
Hi again UmbracoIdentity folk,

OK I did a complete reinstall of Umbraco 7.3.4 but I am still having exactly the same problem :-(

C:\Clancy\A_TeraTastic\Dev\TeraTastic05\TeraTastic\Startup.cs(19,17,19,45): error CS1061: 'IAppBuilder' does not contain a definition for 'UseUmbracoBackAuthentication' and no extension method 'UseUmbracoBackAuthentication' accepting a first argument of type 'IAppBuilder' could be found (are you missing a using directive or an assembly reference?)

So it does not seem to be a pre-requisite issue.

BTW the reason I am trying to include the line  "app.UseUmbracoBackAuthentication();"  is because after I installed UmbracoIdentity I was no longer able to log onto Umbraco BackOffice and I am trying this to see if it resolves that issue.

Any suggestions as to what the problem is or how to solve it would be very much appreciated.

Thanks

Terry Clancy

ClanceZ
